src/current/v23.2/migration-overview.md

```diff
@@ -306,36 +306,48 @@ Using this method, consistency is achieved by only performing the cutover once a
 
-The following is a high-level overview of the migration steps. {% comment %}For details on this migration strategy, see [Migration Strategy: Zero Downtime]({% link {{ page.version.version }}/migration-strategy-lift-and-shift.md %}).{% endcomment %}
+The following is a high-level overview of the migration steps. For details on this migration strategy, see [Migration Strategy: Live Migration]({% link {{ page.version.version }}/migration-strategy-live-migration.md %}).
 
 {% comment %}You can use the CockroachDB Live Migration Service (MOLT LMS) to run application queries simultaneously on your source database and CockroachDB.{% endcomment %}
 
 To prioritize consistency and minimize downtime:
 
-1. {% include {{ page.version.version }}/migration/load-data-third-party.md %} Select the tool's option to **replicate ongoing changes** after performing the initial load of data into CockroachDB. 
+1. {% include {{ page.version.version }}/migration/load-data-third-party.md %} Select the tool's option to **replicate ongoing changes** after performing the initial load of data into CockroachDB.
+
 1. As the data is migrating, you can use [MOLT Verify]({% link {{ page.version.version }}/molt-verify.md %}) to validate the consistency of the data between the source database and CockroachDB.
-1. Once nearly all data from your source database has been moved to CockroachDB (for example, with a <1 second delay or <1000 rows), stop application traffic to your source database. **This begins downtime.**
-1. Wait for replication to CockroachDB to complete.
+
+1. Once nearly all data from your source database has been moved to CockroachDB (for example, with a &lt;1 second delay or &lt;1000 rows), stop application traffic to your source database. **This begins downtime.**
+
+1. Wait for replication to CockroachDB to complete. This typically takes a very short period of time, often less than a few seconds.
+
 1. Perform a [cutover](#cutover-strategy) by resuming application traffic, now to CockroachDB.
 
 To achieve zero downtime with inconsistency:
 
 1. {% include {{ page.version.version }}/migration/load-data-third-party.md %} Select the tool's option to replicate ongoing changes after performing the initial load of data into CockroachDB.
+
 1. As the data is migrating, you can use [MOLT Verify]({% link {{ page.version.version }}/molt-verify.md %}) to validate the consistency of the data between the source database and CockroachDB.
-1. Once nearly all data from your source database has been moved to CockroachDB (for example, with a <1 second delay or <1000 rows), perform a [cutover](#cutover-strategy) by pointing application traffic to CockroachDB.
+
+1. Once nearly all data from your source database has been moved to CockroachDB (for example, with a &lt;1 second delay or &lt;1000 rows), perform a [cutover](#cutover-strategy) by pointing application traffic to CockroachDB.
```

Comment:
there is another approach here that doesn't involve replication, which this section seems to suggest.

* pick a database as a source of truth (e.g. PG)
* do an initial load of data
* write to both databases at the same time and "cutover" using a tool like verify to repair inconsistencies and defaulting back to a "source of truth".
* when you're comfortable, swap which database you consider the source of truth. since the swap here is rarely atomic, you can generate inconsistencies when there is no "central" source of truth. (the "atomic swap" is the tooling that's hard to build)
* fix any inconsistencies from the above time interval, and move on.